pipelines/pipeline_katello_32.yml

```diff
@@ -32,38 +29,36 @@
     - disable_firewall
     - haveged
     - update_os_packages
-    - foreman_installer
 
 - hosts: pipeline-katello-3.2-centos7
   become: yes
   vars:
     foreman_installer_scenario: katello
-    foreman_installer_options_internal_use_only:
+    foreman_installer_options:
       - "--disable-system-checks"
-      - "--foreman-admin-password {{ foreman_installer_admin_password }}"
-    foreman_installer_additional_packages:
-      - katello
+    installer_scenarios_answers:
+      foreman:
+        admin_password: changeme
+    foreman_installer_pkg: katello
   roles:
     - foreman_installer
 
 - hosts: pipeline-proxy-3.2-centos7
   become: yes
   vars:
-    foreman_proxy_content_server: pipeline-katello-3.2-centos7
+    generate_proxy_certs_from: pipeline-katello-3.2-centos7.example.com
```

Comment:
This is missing the `foreman_installer` namespace. This will also break because you are baking in the domain even though the domain is dynamically generated when using Vagrant.